tests/migrations/test_lookup_tables_data_seed.py

```diff
@@ -112,16 +112,169 @@ def test_denied_domains_data_seed(alembic_runner: MigrationContext, alembic_engi
     # Migrate up to, but not including this new migration
     alembic_runner.migrate_up_before("d6e4a13fbebd")
 
+    denied_domains_inserted = [
```

Comment:
Could you alphabetize this list? It'll make it easier to see like-domains and know where to put new ones, rather than always tacking to the end of the list, which makes it more likely to have merge conflicts.

If you're using VSCode, there's a handy **Sort Lines Ascending** command for just this _sort_ (sorry) of thing:

<img width="720" alt="image" src="https://github.com/cfpb/regtech-user-fi-management/assets/4269085/23765a35-83eb-40af-9f0f-34c5b5325825">